_posts/2016-09-16-why-linting-errors-should-cause-tests-to-fail.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Why linting errors should cause tests to fail
+layout: post
+summary: Making the case against allowing linting errors in master.
+---
+
+I recently had a discussion with a colleague who argued that:
+
+> Linting errors should not block a pull request from being merged.
+
+I found myself disagreeing vehemently but unable to clearly state why. I've
+written this post to help clarify my thoughts on the position.
+
+In this post I will attempt to convince you that a linting error should be
```

Comment:
I guess I'm saying that these two paragraphs might read like this "I'm going to write a blog post, which you are about to read. Here it is: (post content)". I believe you can just jump into it, but I might be misreading.